Initializing your Skywallet.md

```diff
@@ -0,0 +1,115 @@
+
+### Step 3.1: First Time Use of a Desktop Wallet
+
+If it is for the first time you are using the desktop wallet then you will see the following screen :
+
+<Screenshot of a Desktop Wallet which is used for the first time>
+
+To configure your Skywallet, proceed to click on **“Using a hardware wallet?”** button.
+
+<Screenshot of a Desktop Wallet highlighting “Using a hardware wallet?” button>
+
+On pressing this button, the display screen on Skywallet will show a welcome message:
+
+<Reusing the Picture showing the Skywallet display with the welcome message>
+
+In order to configure your Skywallet for the first time, you have to install the firmware for the Skywallet, the same is explained in step #4.
+
+### Step 3.2: Previously Configured Desktop Wallet
+
+If you are **not** using your desktop wallet for the first time, then on connecting the Skywallet via USB, the following window will be displayed.
+
+<Screenshot of a previously configured desktop wallet with other wallets as well>
+
+The window will display all your wallets that are already configured using your desktop wallet. (*Note - This may include both Hardware and Software wallets*)
+
+To configure your Skywallet click on the **Hardware Wallet** button.
+
+<Screenshot highlighting the Hardware wallet button along with the context>
+
+On pressing this button, the display screen on Skywallet will show a welcome message:
+
+<Reusing the Picture showing the Skywallet display with the welcome message>
+
+Once the welcome message is displayed on the Skywallet screen, you can proceed to install the firmware.
+
+## Step 4: Installing the Firmware
+
+Once the welcome message is displayed on the Skywallet, the desktop wallet will show the following window:
+
+<Screenshot of the desktop wallet showing the message about the installation of the Firmware>
+
+You can click on continue to start installing the firmware for your Skywallet.
+
+The installation process would take a few seconds to complete. 
+
+Once the installation is complete you can proceed to configure the Skywallet.
+
+**Note - If the installation gets abruptly closed due to any reasons, you can simply disconnect the Skywallet, close the desktop wallet and then retry the whole process from the start.
+
+## Step 4: Skywallet Configuration
+
+Once the firmware installation is complete, the “NEEDS SEED!” message is displayed by the Skywallet.
+
+<Reusing the picture of the Skywallet display showing the message Needs seed>
+
+Simultaneously, the following window will be displayed by the desktop wallet.
+
+<Screenshot of Desktop wallet displaying the message of detecting an unconfigured hardware wallet>
+
+In this window you can select two options :
+
+**Configure automatically**
+
+**Restore backup**
+
+### Step 4.1: Configure automatically
+
+If you are using a Skywallet for the **first** time, then you can proceed with the option of Configure Automatically.
```

Comment:
Be consistent please with "". If you are referencing a button in plain  text, always add the quotes